src/main/java/org/spongepowered/api/world/extent/Extent.java

```diff
@@ -131,6 +132,28 @@ default Vector3i getHighestPositionAt(Vector3i position) {
     }
 
     /**
+     * Returns the y level that precipitation ends falling in the given column.
+     *
+     * <p>A value is still returned for columns in biomes which do not
+     * receive precipitation.</p>
+     *
+     * @return The y level that precipitation ends
+     */
+    int getPrecipitationLevel(int x, int z);
+
+    /**
+     * Returns the y level that precipitation ends falling in the given column.
+     *
+     * <p>A value is still returned for columns in biomes which do not
+     * receive precipitation.</p>
+     *
+     * @return The y level that precipitation ends
+     */
+    default int getPrecipitationLevel(Vector2i column) {
```

Comment:
Well it is nice for consistency and people generally work with Vector3i when working with positions like they would be for this. Sure a Vector2i represents a column well, but it's not really what people are using when they would use this method. I suppose they're generally used for chunks.